build.gradle.kts

```diff
@@ -218,7 +220,7 @@ curseforge {
             mainArtifact(tasks["remapJar"])
 
             relations(closureOf<CurseRelation> {
-                props["cfReqDeps"].toString().split(",").forEach {
+                props["dependencies"].toString().split(",").forEach {
```

Comment:
Why did you change the cf one? It seems to work.

Reply:
It's the same thing for both CF and MR, so there's no point in having the property named `cfReqDeps` when it's being used for MR too.

Reply:
Yeah I just realized that.
Semi related, why doesn't that work on the front end??

Reply:
Does it not?
https://streamable.com/ao31d3

Reply:
Well now I feel crazy again. I swear there was a message that said only project ID